en-US/navigation.lang

```diff
@@ -0,0 +1,302 @@
+## NOTE: https://bedrock-demo-navigation.oregon-b.moz.works/
+## URL: https://bedrock-demo-navigation.oregon-b.moz.works/%LOCALE%/
```

Comment:
The `## URL` here is not particularly useful, since this file doesn't localize a specific page. Once merged, it will point to the root of the dev server. Just leave the demo address in the NOTE, and we'll need to remove it once merged.